confz/loaders/loader.py

```diff
@@ -29,20 +29,21 @@ def update_dict_recursively(cls, original_dict: Dict, update_dict: Dict):
 
     @classmethod
     def transform_nested_dicts(
-        cls, dict_in: Dict[str, Any], separator: str = "."
+        cls, dict_in: Dict[str, Any], separator: str
```

Comment:
Please leave the default value, as o/w we loose backwards compatibility

Reply:
Fixed